src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const body = document.querySelector('body');
```

Comment:
```suggestion
const body = document.body;
```
до боді можна простіше достукатись)